prototype/styles/main.scss

```diff
@@ -1 +1,56 @@
 
+$backgr2: black;
+$fontcolor1: white;
+
+#body{
+    background: url("intro-bg.jpg") no-repeat center center;
+    background-size: cover;
+    filter: sepia(20%);
+}
+
+#maintitle{
+	text-align:center;
+	color:white;
+	margin-top: 60px;
+	margin-bottom:60px;
+}
+#maintitle a{
+	color: $fontcolor1;
+	font-size: 40px;
+	font-weight: bold;
+	letter-spacing: 6px;
+}
+#maintitle a:hover{
+	text-decoration:none;
+}
+#maintitle h2{
+	font-size: 30px;
+	font-weight: bold;
+	letter-spacing: 2px;
+}
+.maintopics{
+	background: $backgr2;
+	opacity: 0.5;
+	height: 150px;
+	padding:5%;
+	text-align: center;
+	font-size: 30px;
+	font-weight: bold;
```

Comment:
Būtų labai gerai jei prisilaikytum css savybių grupavimo: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/